packages/client/src/components/app/table/Table.svelte

```diff
@@ -46,6 +46,13 @@
       selectedRows.map(row => row._id)
     )
   }
+  // Clear row selection if action was trigger
+  $: {
+    if ($rowSelectionStore?.[$component.id]?.clearSelectedRows) {
```

Comment:
I did this because the `$rowSelectionStore` was not being used directly to keep track of the selected rows, and I assumed there was a good reason for that.

Reply:
Not needed, refactoring.